pages/platform/ai-training/14_HOWTO_add_registry/guide.en-gb.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Attach a private registry via UI
-slug: attach-private-registry
-excerpt: Learn how to attach your own registry to AI Training via UI
+title: Add a private registry via UI
+slug: add-private-registry
```

Comment:
As the slug is edited here, should a redirection be added ?

Reply:
If possible, yes